lib/chef/resource_resolver.rb

```diff
@@ -84,9 +69,48 @@ def maybe_dynamic_resource_resolution(resource)
       handlers[0]
     end
 
-    # try the old static lookup of resources by mangling name to resource klass
-    def maybe_chef_platform_lookup(resource)
-      Chef::Resource.resource_matching_short_name(resource)
```

Comment:
Sounds like an excellent plan to me.  I was probably thinking "it seems like all the resolution should actually happen in the resolver," but given that we're deprecating this thing anyway, may as well not care.